packages/localizer/index.ts

```diff
@@ -87,16 +88,16 @@ export const localize = (root?: string) =>
     //@ts-ignore
     CommandInitPlugin(({ module, deps }) => {
         if(module.type === CommandType.Slash || module.type === CommandType.Both) {
-            deps['@sern/logger'].info({ message: "Localizing "+ module.name });
-            const resolvedLocalization= 'command/'+(root??module.name);
-            Reflect.set(module, 'name_localizations', deps.localizer.translationsFor(resolvedLocalization+".name"));
-            Reflect.set(module, 'description_localizations', deps.localizer.translationsFor(resolvedLocalization+'.description'));
-            const newOpts = module.options ?? [];
-            //@ts-ignore 
-            dfsApplyLocalization(newOpts, deps, [resolvedLocalization]);
+            const { localizer, '@sern/logger':log  } = deps
+            const resolvedRoot = 'command/'+(root??module.name);
+            log?.info({ message: "Localizing "+ resolvedRoot });
+            //@ts-ignore
+            dfsApplyLocalization(module.options ?? [], deps, [resolvedRoot]);
+            Reflect.set(module.locals, 'nloc', localizer.translationsFor(resolvedRoot+".name"))
+            Reflect.set(module.locals, 'dloc', localizer.translationsFor(resolvedRoot+'.description'))
             return controller.next();
         } else {
-            //@ts-ignore
+            //@ts-ignore   
             return controller.stop("Cannot localize this type of module " + module.name);
```

Comment:
how's this taking argument?

Reply:
version 4 passes error messages or result payloads now! :D